core/ir/x86/instr_create.h

```diff
@@ -466,6 +466,14 @@
  * \param dc  The void * dcontext used to allocate memory for the instr_t.
  */
 #define XINST_CREATE_nop(dc) INSTR_CREATE_nop(dc)
+
+/**
+ * This platform-independent macro creates an instr_t for an indirect call instr.
+ * \param dc  The void * dcontext used to allocate memory for the instr_t.
+ * \param r   The opnd_t explicit source operand for the instruction. This should
+ * be a reg_id_t operand with the address of the subroutine.
+ */
+#define XINST_CREATE_call_ind(dc, r) INSTR_CREATE_call_ind(dc, r)
```

Comment:
Please add a case to each version of test_xinst()

Reply:
Oops -- didn't notice those tests. Thanks.

It seems that x86 doesn't verify the exact disassembled instr, unlike AArch64. For now, I've maintained the status quo, but if you'd like me to change that in this PR, let me know.

Also, for ARM, as we don't have an ARM machine, I haven't added the expected test result yet. Added a TODO for that.

Reply:
> It seems that x86 doesn't verify the exact disassembled instr

Huh, did not know that.  Would be nice to add it but not high priority: if it encodes and decodes then it should match a real opcode encoding and the other IR tests cover anything else.